src/prepack-standalone.js

```diff
@@ -46,6 +90,7 @@ export function prepack(code: string, options: Options = defaultOptions, errorHa
   return serialized;
 }
 
+/* deprecated: pelase use prepackString instead. */
```

Comment:
Is there a reason this needs to be deprecated? It's not fully implemented because it was a bit tricky to wire up. It would be good to accept a Babel AST since many environments will already have a Babel compatible AST. It's pure overhead to serialize it and parse it again.